contributing.md

```diff
@@ -1,34 +1,15 @@
 # Contribution Guidelines
 
-## Adding to this list
-
 Please ensure your pull request adheres to the following guidelines:
 
 - Search previous suggestions before making a new one, as yours may be a duplicate.
 - Make sure your contribution is useful before submitting.
 - Make an individual pull request for each suggestion.
-- Use the following format: `[Item Name](link)`
+- Use the following format: `[Item Name](link) - Short Description`
 - Link additions should be added in alphabetical order of the relevant category.
 - New categories or improvements to the existing categorization are welcome.
 - Check your spelling and grammar.
 - Make sure your text editor is set to remove trailing whitespace.
 - The pull request and commit should have a useful title.
 
 Thank you for your suggestions!
-
-## Creating your own awesome list
-
-To create your own list, check out the [instructions](create-list.md).
-
-## Adding something to an Awesome list
```

Comment:
Should we keep this for folks not familiar with making pull requests?

I completely agree on all of the other changes. Just not sure about this bottom portion.